tracing/opentracing/client_interceptors.go

```diff
@@ -133,7 +138,18 @@ func newClientSpanFromContext(ctx context.Context, tracer opentracing.Tracer, fu
 	return opentracing.ContextWithSpan(ctxWithMetadata, clientSpan), clientSpan
 }
 
-func finishClientSpan(clientSpan opentracing.Span, err error) {
+func finishClientSpan(ctx context.Context, clientSpan opentracing.Span, err error) {
+	tags := grpc_ctxtags.Extract(ctx)
+	for k, v := range tags.Values() {
```

Comment:
do we want to do this? This will add **all** of the tags that are currently in the context onto the span.